.travis.yml

```diff
@@ -75,6 +76,17 @@ jobs:
         - make image-cephcsi || travis_terminate 1;
         - scripts/travis-functest.sh v1.15.0 || travis_terminate 1;
 
+    - name: cephcsi on Arm64
```

Comment:
default machines provided by Travis will be amd64. i believe this should be amd64 not arm64.

Reply:
Next line "arch: arm64" tells travis to schedule this job to arm64 node.
We can validate it by checking the CI log of this patch: https://travis-ci.org/ceph/ceph-csi/builds/616549765